src/Glpi/Application/ErrorHandler.php

```diff
@@ -328,7 +328,7 @@ public function handleSqlError(int $error_code, string $error_message, string $q
     {
         $this->outputDebugMessage(
             sprintf('SQL Error "%s"', $error_code),
-            sprintf('%s in query "%s"', $error_message, preg_replace('/\\n/', ' ', $query)),
+            sprintf('%s in query "%s"', \htmlspecialchars($error_message), preg_replace('/\\n/', ' ', $query)),
```

Comment:
It will be escaped in `outputDebugMessage` directly.
```suggestion
            sprintf('%s in query "%s"', $error_message, preg_replace('/\\n/', ' ', $query)),
```